sdk/eventhubs/azure_messaging_eventhubs/src/consumer/mod.rs

```diff
@@ -3,6 +3,7 @@
 
 //cspell: words amqp mgmt amqps
 
+#![doc = include_str!("./README.md")]
```

Comment:
Interesting. Was the PWD reference needed? I'm guessing so. I suppose it jives with lib lookup rules on non-Windows platforms.

(Not blocking - just a Q)

Reply:
The PWD reference was *not* needed. I just included it because, well, I pretty much always include it (I have a trust issue with tools that aren't passed in either absolute or relative paths - it's almost always safer to add a relative path in a filename).